debezium-connect-rest-extension/src/test/java/io/debezium/kcrestextension/KafkaConnectResourceActiveIT.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.kcrestextension;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.containsInAnyOrder;
+import static org.hamcrest.Matchers.is;
+
+import java.util.List;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Tests topic creation (which is enabled in Kafka version greater than 2.6.0) and transforms endpoints.
+ * Debezium Container with 1.7 image is used for the same.
+ */
+public class KafkaConnectResourceActiveIT {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(KafkaConnectResourceActiveIT.class);
+    private static int EXPOSED_PORT;
+
+    @BeforeEach
+    public void start() {
+        TestHelper.stopContainers();
+        TestHelper.setupDebeziumContainer("1.7");
+        TestHelper.startContainers();
+
+        EXPOSED_PORT = TestHelper.getDebeziumContainer().getFirstMappedPort();
+    }
+
+    @AfterEach
+    public void stop() {
+        TestHelper.stopContainers();
+    }
+
+    @Test
+    public void testTopicCreationEndpoint() {
```

Comment:
This class should not be called `...ActiveIT` as here's still the test case missing for Kafka >= 2.6.0 but `topic.creation.enable = false`.
Can you find a better name, for example just call this one `DebeziumResourceIT`, and the other one `DebeziumResourceNotActiveIT` or something so they have a relation to the tested classname.
And add this test to this test class with 1.7?
Also, maybe you want to make `1.7` a constant in the class so it's easier to update to a newer version? Or can we maybe derive it from Maven?